### ISSUE TITLE
is there persistence 

### ISSUE THREAD

**Issue by jigsaw678** [Seytonic/Duckduino-microSD#15]
if once the victim machine is injected by the payload and they close their machine and turn it back on will the payload still come back online?

**Comment by AshutoshGeek** [comment]
@jigsaw678 No, it will not because the payload is executed by the arduino only when it is plugged in.
